modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -79,7 +80,8 @@ export default {
       disabled: false,
       searching: false,
       choosing: false,
-      relationshipSchema: null
+      relationshipSchema: null,
+      imageRelationshipComponent: 'AposImageRelationshipEditor'
```

Comment:
This shouldn't be hardcoded browser side, the purpose of the server side configuration of component names is to allow injection of alternatives appropriate to specific relationships.